waspc/src/Wasp/AppSpec.hs

```diff
@@ -14,5 +15,9 @@ data AppSpec = AppSpec
   { -- | List of declarations like App, Page, Route, ... that describe the web app.
     decls :: [Decl],
     -- | List of external code files (they are referenced/used by the declarations).
-    externalCodeFiles :: ExternalCode.File
+    externalCodeFiles :: [ExternalCode.File],
+    -- | Absolute path to the directory in wasp project source that contains external code files.
+    externalCodeDirPath :: !(Path' Abs (Dir ExternalCode.SourceExternalCodeDir)),
+    dotEnvFile :: Maybe (Path' Abs File'),
+    isBuild :: Bool
```

Comment:
We recently added `migrationsDir` as well: https://github.com/wasp-lang/wasp/blob/master/waspc/src/Wasp/Wasp.hs#L64 I think we may need that too.

Reply:
Thanks @shayneczyzewski you are right, I missed that one! Added it now.